http/src/main/java/com/github/awsjavakit/http/TokenProvider.java

```diff
@@ -24,13 +28,26 @@ static TokenProvider locallyCachedTokenProvider(
     OAuthCredentialsProvider authCredentialsProvider,
     Duration maxTokenAge) {
     var tokenRefresher =
-      NewTokenProvider.create(httpClient,authCredentialsProvider);
-    return new CachedTokenProvider(tokenRefresher,maxTokenAge);
+      NewTokenProvider.create(httpClient, authCredentialsProvider);
+    return new CachedTokenProvider(tokenRefresher, maxTokenAge);
   }
 
+  static TokenCacheUpdateStrategy<OAuthTokenEntry> defaultUpdateStrategy() {
```

Comment:
TokenProvider interface contains methods for the defaultUpdateStrategy and the different variants for TokenProviders